feat(node): Add docs for `NODE_OPTIONS` esm config (#10093)
getsentry/sentry-docs: docs/platforms/javascript/common/install/esm.mdx

node --import ./instrument.mjs app.mjs
```

> Sometimes, you cannot directly pass the `--import` flag to the Node.js binary. In such cases, you can use the `NODE_OPTIONS` environment variable to pass it, like this:

Reviewer: ```suggestion
If it is not possible for you to pass the `--import` flag to the Node.js binary, you can alternatively use the `NODE_OPTIONS` environment variable as follows:
```
